apps/condo/migrations/20230424210801-0263_incidentexporttaskhistoryrecord_incidentexporttask.js

```diff
@@ -0,0 +1,44 @@
+
+exports.up = async (knex) => {
+    await knex.raw(`
+    BEGIN;
+--
+-- Create model incidentexporttaskhistoryrecord
+--
+CREATE TABLE "IncidentExportTaskHistoryRecord" ("status" text NULL, "format" text NULL, "exportedRecordsCount" integer NULL, "totalRecordsCount" integer NULL, "file" jsonb NULL, "meta" jsonb NULL, "where" jsonb NULL, "sortBy" jsonb NULL, "locale" text NULL, "timeZone" text NULL, "user" uuid NULL, "id" uuid NOT NULL PRIMARY KEY, "v" integer NULL, "createdAt" timestamp with time zone NULL, "updatedAt" timestamp with time zone NULL, "createdBy" uuid NULL, "updatedBy" uuid NULL, "deletedAt" timestamp with time zone NULL, "newId" jsonb NULL, "dv" integer NULL, "sender" jsonb NULL, "history_date" timestamp with time zone NOT NULL, "history_action" varchar(50) NOT NULL, "history_id" uuid NOT NULL);
+--
+-- Create model incidentexporttask
+--
+CREATE TABLE "IncidentExportTask" ("status" varchar(50) NOT NULL, "format" varchar(50) NOT NULL, "exportedRecordsCount" integer NOT NULL, "totalRecordsCount" integer NOT NULL, "file" jsonb NULL, "meta" jsonb NULL, "where" jsonb NOT NULL, "sortBy" jsonb NOT NULL, "locale" text NOT NULL, "timeZone" text NOT NULL, "id" uuid NOT NULL PRIMARY KEY, "v" integer NOT NULL, "createdAt" timestamp with time zone NULL, "updatedAt" timestamp with time zone NULL, "deletedAt" timestamp with time zone NULL, "newId" uuid NULL, "dv" integer NOT NULL, "sender" jsonb NOT NULL, "createdBy" uuid NULL, "updatedBy" uuid NULL, "user" uuid NOT NULL);
+CREATE INDEX "IncidentExportTaskHistoryRecord_history_id_899d7c7a" ON "IncidentExportTaskHistoryRecord" ("history_id");
+ALTER TABLE "IncidentExportTask" ADD CONSTRAINT "IncidentExportTask_createdBy_0a6be750_fk_User_id" FOREIGN KEY ("createdBy") REFERENCES "User" ("id") DEFERRABLE INITIALLY DEFERRED;
+ALTER TABLE "IncidentExportTask" ADD CONSTRAINT "IncidentExportTask_updatedBy_27aa8f50_fk_User_id" FOREIGN KEY ("updatedBy") REFERENCES "User" ("id") DEFERRABLE INITIALLY DEFERRED;
+ALTER TABLE "IncidentExportTask" ADD CONSTRAINT "IncidentExportTask_user_0ccaac00_fk_User_id" FOREIGN KEY ("user") REFERENCES "User" ("id") DEFERRABLE INITIALLY DEFERRED;
+CREATE INDEX "IncidentExportTask_createdAt_bced5491" ON "IncidentExportTask" ("createdAt");
+CREATE INDEX "IncidentExportTask_updatedAt_6d9d1118" ON "IncidentExportTask" ("updatedAt");
+CREATE INDEX "IncidentExportTask_deletedAt_9ed50375" ON "IncidentExportTask" ("deletedAt");
+CREATE INDEX "IncidentExportTask_createdBy_0a6be750" ON "IncidentExportTask" ("createdBy");
+CREATE INDEX "IncidentExportTask_updatedBy_27aa8f50" ON "IncidentExportTask" ("updatedBy");
+CREATE INDEX "IncidentExportTask_user_0ccaac00" ON "IncidentExportTask" ("user");
+COMMIT;
```

Comment:
please make sure you've added all indexes on all fields that could be used to sort/filter data